### ISSUE TITLE
Trouble with event detection

### ISSUE THREAD

**Issue by ccleaud** [jcaillon/3P#32]
Hi,

Considering this piece of code:
```
ON END-ERROR OF C-Win /* Errors Listing */
OR ENDKEY OF {&WINDOW-NAME} ANYWHERE DO:
    RETURN NO-APPLY.
END.
```

The code explorer finds:
* END-ERROR C-Win [ OK ]
* END-ERROR, ENDKEY C-Win, {&WINDOW-NAME} [ NOT OK ]

Thanks for fixing.

Regards,
C.CLEAUD

**Comment by ccleaud** [comment]
More over, clicking on both elements goes to the same line.

**Comment by jcaillon** [comment]
`More over, clicking on both elements goes to the same line.`
this is intended, it goes to the definition line i.e. the line of the statement; since it is one statement, it goes to the same line. I won't change this behavior :)

The other part is fixed in v1.1.0

**Comment by ccleaud** [comment]
I won't change this behavior :) --> I'm pretty OK!
